src/main/java/gov/nasa/jpl/mbee/mdk/mms/actions/ValidateElementDepthAction.java

```diff
@@ -1,85 +0,0 @@
-package gov.nasa.jpl.mbee.mdk.mms.actions;
```

Comment:
This removal is not desired as the working plan is to address the performance issue on the server-side. This functionality is desired and, as far as I am aware, still working on models that are sufficiently small to not trigger the performance issue. A transparent fix on the server-side to accommodate larger models would allow this to work unchanged. Should the fix not be transparent, then we can re-implement it based on the new server spec once it is available.